nil/internal/execution/state.go

```diff
@@ -1225,6 +1227,9 @@ func decodeRevertTransaction(data []byte) string {
 	var revString string
 	if index := bytes.IndexByte(data, 0); index > 0 {
 		revString = string(data[:index])
+		if !utf8.ValidString(revString) {
+			return hexutil.Encode(data[:index])
```

Comment:
Please add some prefix for better understanding that it is not a utf string.
```suggestion
			return "Not a UTF string: "+hexutil.Encode(data[:index])
```